projects/angular/src/forms/combobox/option.ts

```diff
@@ -78,8 +78,9 @@ export class ClrOption<T> implements OnInit {
     }
   }
 
-  @HostListener('click')
-  onClick() {
+  @HostListener('click', ['$event'])
+  onClick(event) {
```

Comment:
```suggestion
  onClick(event: MouseEvent) {
```

Reply:
Added in [ed14c8a](https://github.com/vmware-clarity/ng-clarity/pull/1540/commits/ed14c8a6dc53d57bc5d0ac4fa65096fa323d915b)